src/otx/recipe/_base_/train.yaml

```diff
@@ -5,8 +5,9 @@ callbacks:
     init_args:
       monitor: null
       mode: max
-      patience: 100
+      patience: 10
```

Comment:
Shouldn't we also raise `max_epochs`?

Reply:
At the regression tests, we've already set the proper epochs. However, set the proper epochs as a default is important. However, it needs a branch for different tasks. Let's consider this at the later PR.